vllm/model_executor/layers/fused_moe/__init__.py

```diff
@@ -1,17 +1,23 @@
+from vllm.model_executor.layers.fused_moe.fused_moe_marlin import (
+    fused_moe_marlin, single_moe_marlin)
 from vllm.model_executor.layers.fused_moe.layer import (
     FusedMoE, FusedMoEMethodBase, FusedMoeWeightScaleSupported)
 from vllm.triton_utils import HAS_TRITON
 
-__all__ = ["FusedMoE", "FusedMoEMethodBase", "FusedMoeWeightScaleSupported"]
+__all__ = [
+    "FusedMoE",
+    "FusedMoEMethodBase",
+    "FusedMoeWeightScaleSupported",
+    "fused_moe_marlin",
+    "single_moe_marlin",
```

Comment:
`fused_moe_marlin` and `single_moe_marlin` wil have to be moved to the `HAS_TRITON` condition. Should help with the failing tpu test